### PR TITLE
Playing Accessible Media

### PR DESCRIPTION
**Pull request by yatil**
Comments in the PR.

### CODE REVIEW

File: source/accessible-media/playing.html.erb.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Playing accessible media
-nav_title: Playing media
+title: Playing Accessible Media
```
Comment (yatil) [inline source/accessible-media/playing.html.erb.md:2]: Title case.

File: source/accessible-media/playing.html.erb.md [hunk 2]
```diff
@@ -18,29 +18,31 @@ support: Developed with support from the <a href="https://www.w3.org/WAI/WCAGTA/
...
-render timed text delivered via the `track` element. All browsers will
+All modern web browsers will provide default media players for embedded
+media when the `<video>` or `<audio>` elements are used, and all will
+render timed text delivered via the `<track>` element.
+
+All browsers will
 automatically surface a caption toggle in the player control bar if a
-caption track is detected. However, note that each browser will present
+caption track is detected. Note that each browser will present
 not only a different UI, but also a different default style for
```
Comment (yatil) [inline source/accessible-media/playing.html.erb.md:28]: This sounds like a negative, however having a similar experience in the browser of choice seems better to me than every site doing their own UI.

File: source/accessible-media/playing.html.erb.md [hunk 3]
```diff
@@ -18,29 +18,31 @@ support: Developed with support from the <a href="https://www.w3.org/WAI/WCAGTA/
...
-captions. Below is an image of default caption display in Chrome,
-showing the CC button for toggling captions on and off.
+captions. Users will customize captions in their browsers or operating systems to their needs. Below is an image of default caption display in Chrome.
+Captions can be toggled on and off by using the CC (Closed Captions) button.
```
Comment (yatil) [inline source/accessible-media/playing.html.erb.md:30]: Describing what CC stands for.

File: source/accessible-media/playing.html.erb.md [hunk 4]
```diff
@@ -18,29 +18,31 @@ support: Developed with support from the <a href="https://www.w3.org/WAI/WCAGTA/
...
 
 -   **Android**<br>
     Embedded video and audio clips will play within Web pages in both
     Chrome and Firefox, on tablets and smartphones.
 -   **iOS**<br>
     Embedded video and audio clips will play within Web pages on the
-    iPad. iPhones will play all media clips using the QuickTime player.
+    iPad. iPhones will play all media clips in full screen.
 
 ## Audio descriptions in native desktop- and mobile-browser media players
```
Comment (yatil) [inline source/accessible-media/playing.html.erb.md:47]: This section repeats a section from the audio description page but does not explain what you need to do to play an audio file with audio description. If we can’t say something concrete, we should not list this.

File: source/accessible-media/playing.html.erb.md [hunk 5]
```diff
@@ -59,19 +61,19 @@ following methods:
...
 users can specify caption-font colors, font types and sizes, and
-background colors or transparency levels. Note that in any player on any
-operating system, the settings chosen by the user will override any
-author-specified styling characteristics (see further [comments about
+background colors or transparency levels.
+The settings chosen by the user will override any
+author-specified styling (see further [comments about
 author styling of captions](production-captions.html#a-word-about-styling-captions)).
 
-Below is an image showing caption-styling options in the YouTube desktop
```
Comment (yatil) [inline source/accessible-media/playing.html.erb.md:67]: I don't think which player it is matters here.

File: source/accessible-media/playing.html.erb.md [hunk 6]
```diff
@@ -59,19 +61,19 @@ following methods:
...
 
 ![YouTube's caption-styling options: font family, font color, font size,
 and others.](yt_cc_options.png)
 
-Some custom mobile players-- those embedded in Web pages as well as
-stand-alone media players-- also provide styling options from within the
+Some custom mobile players – those embedded in Web pages as well as
+stand-alone media players – also provide styling options from within the
 applications themselves. See the [mobile player used on the CNN Web
 site](http://www.cnn.com/) for one such example.
```
Comment (yatil) [inline source/accessible-media/playing.html.erb.md:78]: Just pointing at CNN.com is confusing. Where should I click, which player is meant?  I don’t think we want to have a link to a resource that might change its implementation over night.

I’d also feel better with a non-commercial example and best with an inline example.

Comment (gfreedncam) [reply source/accessible-media/playing.html.erb.md:78]: pointed to the cnn video page and added instructions for playing videos with captions.

File: source/accessible-media/playing.html.erb.md (hunk 5 above)
Comment (gfreedncam) [reply source/accessible-media/playing.html.erb.md:67]: fixed.

File: source/accessible-media/playing.html.erb.md (hunk 4 above)
Comment (gfreedncam) [reply source/accessible-media/playing.html.erb.md:47]: fixed.

File: source/accessible-media/playing.html.erb.md (hunk 3 above)
Comment (gfreedncam) [reply source/accessible-media/playing.html.erb.md:30]: fixed.

File: source/accessible-media/playing.html.erb.md (hunk 2 above)
Comment (gfreedncam) [reply source/accessible-media/playing.html.erb.md:28]: fixed.

File: source/accessible-media/playing.html.erb.md (hunk 1 above)
Comment (gfreedncam) [reply source/accessible-media/playing.html.erb.md:2]: fixed (will fix on other pages for consistency).

**Comment by gfreedncam** [comment]
addressed items above.

**Pull request closed without merging**